critiquebrainz/frontend/templates/macros.html

```diff
@@ -67,6 +67,13 @@
         {{ _('Label') }}
       </span>
     {% endif %}
+  {% elif entity_type == 'recording'  %}
+    <img src="{{ get_static_path('images/placeholder_disc.svg') }}" {{ attributes }} />
+    {% if overlay_type %}
+      <span class="entity-type recording">
+        {{ _('Recording') }}
+      </span>
+    {% endif %}
   {% else %} {# release-group #}
```

Comment:
it'd be nice to work out how to remove this fall-through else here, and require this to be called with entity_type="release-group" in those situations

Reply:
I checked it is being called with `release_group` already elsewhere so simply changing else to elif was enough.